<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtTask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveAll">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
